services/web-app/lib/github.js

```diff
@@ -806,6 +789,42 @@ const getLibraryAssets = async (params = {}) => {
   return assets
 }
 
+/**
+ * Validates and filters an array of assets,
+ * returns a modified array containing only valid assets for a library.
+ * @param {Promise<import('../typedefs').Library>} library
+ */
+const validateLibraryAssets = (library) => {
+  library.assets = library.assets.filter((asset) => {
+    const isValidAsset = validateAsset(asset.content, library)
+    if (library.params.asset) {
+      return isValidAsset && getSlug(asset.content) === library.params.asset
+    }
+    return isValidAsset
+  })
+}
+
+/**
+ * Adds inherited data to library.
+ * @param {import('../typedefs').Library} library
+ * @returns {Promise<void>}
```

Comment:
Does this return anything or just modify the param?

Reply:
Remove the `@returns`?

Reply:
it does return a promise that resolves to undefined, we are awaiting the function when called, I think the @returns notation of an empty promise makes sense here. Double check me on that @jdharvey-ibm ?

Reply:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/51208233/186762865-76d18cc5-e63b-49c2-b276-a1758ceb844c.png">

VS Code agrees! The docs look a little funny not having any descriptive text after the return value though. Maybe just add `@returns {Promise<void>} A promise that resolves to void.`?

Reply:
Done!